src/cobra/flux_analysis/deletion.py

```diff
@@ -405,3 +405,47 @@ def double_gene_deletion(
         processes=processes,
         **kwargs
     )
+
+
+@pd.api.extensions.register_dataframe_accessor("knockout")
+@pd.api.extensions.register_series_accessor("knockout")
+class KnockoutAccessor:
+    """Access unique combinations of reactions in deletion results."""
+
+    def __init__(self, pandas_obj):
+        """Set up the accessor.
+
+        Parameters:
+        -----------
+        pandas_obj : pd.DataFrame or pd.Series
+            A result from one of the deletion methods.
+        """
+        self._validate(pandas_obj)
+        self._result = pandas_obj
+
+    @staticmethod
+    def _validate(obj):
+        # verify it is a deletion results
+        if obj.index.name != "ids":
+            raise AttributeError("Must be DataFrame returned by a deletion method.")
+
+    def __getitem__(self, args):
```

Comment:
```suggestion
    def __getitem__(self, args: typing.Iterable[typing.Union[str, Gene, Reaction]]) -> pd.Series:
```

Reply:
Why should row indexing on a DataFrame return a Series?

Reply:
My bad, I was going off of `.iterrows` behaviour but it indeed returns a data frame with a single row.

Reply:
That's weird behaviour, I just played with a MultiIndex and there the returned object is indeed a series.